### PR TITLE
fix(main): fix rootfs struct spilt clean shell

### PR DESCRIPTION
**Pull request by cuisongliu**

### CODE REVIEW

File: filesystem/rootfs/docker/scripts/clean.sh [hunk 1]
```diff
@@ -1,35 +1,7 @@
 #!/bin/bash
 
-systemctl stop docker
-rm -rf /etc/docker/daemon.json
-rm -rf /lib/systemd/system/docker.service
-rm -rf /usr/lib/systemd/system/docker.service
-systemctl daemon-reload
+chmod a+x clean-kube.sh
+chmod a+x clean-docker.sh
```
Comment (oldthreefeng) [inline filesystem/rootfs/docker/scripts/clean.sh:4]: It is good to separate docker and kubelet in clean progress.

File: filesystem/rootfs/docker/scripts/clean-docker.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+systemctl daemon-reload
+
+rm -f /usr/bin/containerd
+rm -f /usr/bin/containerd-shim
+rm -f /usr/bin/ctr
+rm -f /usr/bin/docker
+rm -f /usr/bin/docker-init
+rm -f /usr/bin/docker-proxy
+rm -f /usr/bin/dockerd
+rm -f /usr/bin/runc
```
Comment (oldthreefeng) [inline filesystem/rootfs/docker/scripts/clean-docker.sh:16]: we need to find out all of the binaries install by sealer and delete it all. 
i'm not sure  `/usr/bin/containerd-shim-runc-v2` should be list in here ?

**Review by oldthreefeng — COMMENTED** [review]
LGTM

File: filesystem/rootfs/docker/scripts/clean-docker.sh (hunk 2 above)
Comment (cuisongliu) [reply filesystem/rootfs/docker/scripts/clean-docker.sh:16]: at newest image `/usr/bin/containerd-shim-runc-v2`  already not exist

**Review by fanux — APPROVED** [review]

**Pull request merged**